packages/front-end/components/Experiment/ExperimentMetricTimeSeriesGraphWrapper.tsx

```diff
@@ -171,3 +185,18 @@ export default function ExperimentMetricTimeSeriesGraphWrapper({
     />
   );
 }
+
+function Message({ children }: { children: React.ReactNode }) {
+  return (
+    <Flex
+      align="center"
+      height="220px"
```

Comment:
Should we make it smaller? Probably awkward if loading is big and then it shrinks smaller for error state?

Reply:
All of the states would have the same height because the graph is 220px, this way we avoid any layout change (besides the expansion of the drawer).